config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'welcome#index'
+
+  get 'quotes', to: 'quotes#search', as: 'quotes_search'
```

Comment:
We are indeed searching for quotes but if we are thinking in a RESTful manner, when we hit that `search` button we are being taken to a quote index page. So why did you choose to use a non-RESTful route? :)